.travis.yml

```diff
@@ -5,8 +5,9 @@ python:
     - 2.7
     - 3.3
     - 3.4
+    - pypy
```

Comment:
Added. There was also a library incompatibility, because PyPy3 implements Python3.2. Unfortunately, this doesn't let us add CPython3.2, because PyPy3 supports the `u""` syntax, but CPython3.2 doesn't, and the `zope.publisher` dependency uses that.